### PR TITLE
Game info

### PR DESCRIPTION
**Pull request by AmberMLiu**
Add select game use case and game info entity.

### CODE REVIEW

File: src/main/java/interface_adapter/game_setup/GameSetupController.java
Comment (EstelleWang-UofT) [inline src/main/java/interface_adapter/game_setup/GameSetupController.java:1]: shouldn't the controller implement the input boundary?

Comment (EstelleWang-UofT) [reply src/main/java/interface_adapter/game_setup/GameSetupController.java:1]: No actually, I think the controller shouldn't be an implementation of any interface.

**Review by EstelleWang-UofT — CHANGES_REQUESTED** [review]
Minor changes should be made. Please see my comment on GameSetupController.java .
I am also worried about the TODOs you wrote. Maybe finish them before merging into main?
If the TODOs are assigned to another person, they can also make changes to this Game-info big part of the code.

**Comment by AmberMLiu** [comment]
Fixed the GameSetupController problem, and implemented one TODO. Other TODOs are not related to this feature, I placed them there as a place holder for other people.

**Review by EstelleWang-UofT — APPROVED** [review]

**Pull request merged**